### PR TITLE
Icon invalid value

### PR DESCRIPTION
**Pull request by joakim-hove**
See: #395

### CODE REVIEW

File: lib/ecl/well_conn.cpp [hunk 1]
```diff
@@ -157,73 +154,85 @@ well_conn_type * well_conn_alloc_from_kw( const ecl_kw_type * icon_kw ,
...
+  well_conn_dir_enum dir = well_conn_fracX;
+
+  /* Set the status */
+  {
+    int int_status = ecl_kw_iget_int( icon_kw , icon_offset + ICON_STATUS_INDEX );
+    if (int_status > 0)
+      open = true;
+    else
+      open = false;
+  }
```
Comment (pgdr) [inline lib/ecl/well_conn.cpp]: These lines could perhaps be replaced with
```c
bool open = ecl_kw_iget_int(icon_kw, icon_offset + ICON_STATUS_INDEX) > 0
```

File: lib/ecl/well_conn.cpp [hunk 2]
```diff
@@ -157,73 +154,85 @@ well_conn_type * well_conn_alloc_from_kw( const ecl_kw_type * icon_kw ,
...
+    break;
+  case(ICON_FRACX):
+    dir = well_conn_fracX;
+    break;
+  case(ICON_FRACY):
+    dir = well_conn_fracY;
+    break;
+  default:
+    util_abort("%s: icon direction value:%d not recognized\n",__func__ , int_direction);
+  }
```
Comment (pgdr) [inline lib/ecl/well_conn.cpp]: You can remove `default` now if we have the outside range covered above?

File: lib/ecl/well_conn.cpp (hunk 1 above)
Comment (pgdr) [reply lib/ecl/well_conn.cpp]: Perhaps we should rename `open` to something else

Comment (joakim-hove) [reply lib/ecl/well_conn.cpp]: Have replaced the `open` assignment with your suggestion. The name `open` is quite descriptive in my opinion - so I have left that.

File: lib/ecl/well_conn.cpp (hunk 2 above)
Comment (joakim-hove) [reply lib/ecl/well_conn.cpp]: Done.

**Comment by joakim-hove** [comment]
Thank you for the review

File: lib/ecl/well_conn.cpp (hunk 1 above)
Comment (pgdr) [reply lib/ecl/well_conn.cpp]: The point wasn't that it wasn't descriptive, but I thought maybe it wasn't considered in good taste to reuse standard library names.

Comment (joakim-hove) [reply lib/ecl/well_conn.cpp]: OK - I agree; have changed to `is_open`

**Pull request merged**